### ISSUE TITLE
Keyboard Won't Pop-up

### ISSUE THREAD

**Issue by Crafty19** [AOF-Dev/MCinaBox#763]
I can't type, cause my keyboard phone won't pop-up. Please help.

 - Device: [Realme 5i]
 - App Version [v0.1.4-p4]

**Comment by ThatGifIsNice** [comment]
Same for me

**Comment by ThatGifIsNice** [comment]
Same for me

**Comment by GreenninjaGamingyt** [comment]
> I can't type, cause my keyboard phone won't pop-up. Please help.
> 
>  - Device: [Realme 5i]
>  - App Version [v0.1.4-p4]
> 

Well its different now To type you need to turn on pc keyboard there is a button of enable type click it then using it you can type but you must click on place where you are typing

**Comment by ThatGifIsNice** [comment]
> > I can't type, cause my keyboard phone won't pop-up. Please help.
> > 
> > * Device: [Realme 5i]
> > * App Version [v0.1.4-p4]
> 
> Well its different now To type you need to turn on pc keyboard there is a button of enable type click it then using it you can type but you must click on place where you are typing

Hey do you know how to move that keyboard or the moving joystick because it seems like i cant

**Comment by GreenninjaGamingyt** [comment]
> > > I can't type, cause my keyboard phone won't pop-up. Please help.
> > > 
> > > * Device: [Realme 5i]
> > > * App Version [v0.1.4-p4]
> > 
> > Well its different now To type you need to turn on pc keyboard there is a button of enable type click it then using it you can type but you must click on place where you are typing
> 
> Hey do you know how to move that keyboard or the moving joystick because it seems like i cant

Yes there is a lock sign there click on it then use transparent area of keyboard to move it

**Comment by Crafty19** [comment]
> > I can't type, cause my keyboard phone won't pop-up. Please help.
> > 
> > * Device: [Realme 5i]
> > * App Version [v0.1.4-p4]
> 
> Well its different now To type you need to turn on pc keyboard there is a button of enable type click it then using it you can type but you must click on place where you are typing

tq but how to move it?

**Comment by Crafty19** [comment]
> > > > I can't type, cause my keyboard phone won't pop-up. Please help.
> > > > 
> > > > * Device: [Realme 5i]
> > > > * App Version [v0.1.4-p4]
> > > 
> > > 
> > > Well its different now To type you need to turn on pc keyboard there is a button of enable type click it then using it you can type but you must click on place where you are typing
> > 
> > 
> > Hey do you know how to move that keyboard or the moving joystick because it seems like i cant
> 
> Yes there is a lock sign there click on it then use transparent area of keyboard to move it

ohhh... nice nixce
